sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3145,7 +3145,9 @@ object SQLConf {
       DeprecatedConfig(AVRO_REBASE_MODE_IN_WRITE.alternatives.head, "3.2",
         s"Use '${AVRO_REBASE_MODE_IN_WRITE.key}' instead."),
       DeprecatedConfig(AVRO_REBASE_MODE_IN_READ.alternatives.head, "3.2",
-        s"Use '${AVRO_REBASE_MODE_IN_READ.key}' instead.")
+        s"Use '${AVRO_REBASE_MODE_IN_READ.key}' instead."),
+      DeprecatedConfig(LEGACY_REPLACE_DATABRICKS_SPARK_AVRO_ENABLED.key, "3.2",
+        """Use `.format("avro")` in `DataFrameWriter` or `DataFrameReader` instead.""")
```

Comment:
Ur, is this guide correct?

Reply:
The configs is about automatically mapping of `.format("com.databricks.spark.avro")` to `.format("avro")`, right? If we remove the config in the future, "com.databricks.spark.avro" will be not mapped to built in `avro`. So, in the guide, we recommend to change users code, and use the `avro` format directly.

Reply:
Thanks. I was confused at that.